vision/getting-started/image_editing.ipynb

```diff
@@ -1,710 +1,1247 @@
 {
```

Comment:
<p>Change "superior" to "better".</p><p> </p><p>Change "trick" to "tip".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Changed "special case" to tip and superior to better. Changed trick to method. </p>